### PR TITLE
Update readme

### LINKED ISSUE #47: Update the readme
- [ ] Describe supported WordPress versions (minimum 5.2), add badges
- [ ] Describe how to use the `run-all-cores.sh` script

### PR DESCRIPTION
**Pull request by cadic**
### Description of the Change
Updated the readme:
- [x] Shields
- [x] Anchor text for the setup link
- [x] Docs for `run-all-cores.sh`
- [x] Work with us banner

Closes #47

### CODE REVIEW

**Review by iamdharmesh — APPROVED** [review]
@cadic Thanks for adding this. LGTM!!

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # cypress-wp-utils
 
+![WordPress latest](https://img.shields.io/badge/WordPress%20up%20to-6.0-blue) ![WordPress minimum](https://img.shields.io/badge/WordPress%20since-5.2-blue) ![Test PRs](https://github.com/10up/cypress-wp-utils/actions/workflows/cypress.yml/badge.svg) ![CodeQL](https://github.com/10up/cypress-wp-utils/actions/workflows/codeql-analysis.yml/badge.svg)
+
 > Utilities library for WordPress E2E testing in the Cypress environment.
 
 ## Prerequisites
 
-This library requires Cypress. Use https://github.com/10up/cypress-wp-setup to set up Cypress automatically.
+This library requires Cypress. Use [@10up/cypress-wp-setup](https://github.com/10up/cypress-wp-setup) to set up Cypress automatically.
```
Comment (dinhtungdu) [inline README.md]: @cadic because `@10up/cypress-wp-setup` now includes this library, do you think we should note that behavior here?

Comment (jeffpaul) [reply README.md]: @cadic bumping this question for you^

Comment (cadic) [reply README.md]: @dinhtungdu @jeffpaul added the comment about the current library included into the setup.

**Pull request merged**